src/coreclr/runtime/loongarch64/AllocFast.S

```diff
@@ -216,20 +184,37 @@ LOCAL_LABEL(ArraySizeOverflow):
         ori  $a1, $zero, 1 // Indicate that we should throw OverflowException
         b  C_FUNC(RhExceptionHandling_FailedAllocation)
 
-LOCAL_LABEL(RhpNewArray_Rare):
-        b  C_FUNC(RhpNewArrayRare)
-    LEAF_END    RhpNewArray, _TEXT
+    LEAF_END    RhpNewArrayFast, _TEXT
+
+#ifndef FEATURE_NATIVEAOT
+// Allocate one-dimensional, zero-based array (SZARRAY) of objects (pointer sized elements).
+//  $a0 == MethodTable
+//  $a1 == element count
+    LEAF_ENTRY RhpNewObjectArrayFast, _Text
+
+        li.w  $a2, ((ASM_LARGE_OBJECT_SIZE - 256)/8) // sizeof(void*)
```

Comment:
Is this an alternative name defined by LA's assembler? `li.w` is only used for 1 place in `context2.S`. The majority is using `ori $dest, $zero, imm` to load a small constant.

Reply:
It's a pseduo-instruction that should expand to correct pattern based on the size of the constant.